src/metabase/api/embed.clj

```diff
@@ -237,10 +237,10 @@
 (defn card-for-unsigned-token
   "Return the info needed for embedding about Card specified in `token`. Additional `constraints` can be passed to the
   `public-card` function that fetches the Card."
-  {:style/indent 1}
   [unsigned-token & {:keys [embedding-params constraints]}]
-  (let [card-id        (embed/get-in-unsigned-token-or-throw unsigned-token [:resource :question])
-        token-params   (embed/get-in-unsigned-token-or-throw unsigned-token [:params])]
+  {:pre [((some-fn empty? sequential?) constraints) (even? (count constraints))]}
```

Comment:
This was being called like

```clj
(card-for-unsigned-token <token> :constraints [[:x 1]])
```

which was passed to Toucan as something like

```clj
(db/select Card :token <token> [:x 1])
```

This was never supposed to work, but somehow it did in Toucan 1. In Toucan 2, I don't plan on making this work, since the original behavior was undocumented an unintentional. I changed things to ensure we execute something like

```clj
(db/select Card :token <token> [x 1)
```

one instead which is the equivalent Toucan 2-compatibile version.